specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-01-15-preview/failoverGroups.json

```diff
@@ -0,0 +1,231 @@
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FailoverGroupResource"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "FailoverGroupResource": {
+      "type": "object",
+      "description": "Failover group resource",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FailoverGroupProperties",
+          "description": "null",
+          "x-ms-client-flatten": false
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "FailoverGroupProperties": {
+      "type": "object",
+      "description": "The properties of a failover group resource.",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Accepted"
+          ],
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          },
+          "description": "The provisioning state of the failover group resource.",
+          "readOnly": true
+        },
+        "partnerManagedInstanceId": {
+          "type": "string",
+          "description": "The resource ID of the partner SQL managed instance."
+        },
+        "spec": {
+          "$ref": "#/definitions/FailoverGroupSpec",
+          "description": "null",
+          "x-ms-client-flatten": false
+        },
+        "status": {
+          "$ref": "#/definitions/FailoverGroupStatus",
+          "description": "null",
+          "x-ms-client-flatten": false
+        }
+      },
+      "required": [
+        "partnerManagedInstanceId",
+        "spec"
+      ]
+    },
+    "FailoverGroupSpec": {
+      "description": "The specifications of the failover group resource.",
+      "type": "object",
+      "properties": {
+        "sharedName": {
+          "type": "string",
+          "description": "The shared name of the failover group for this SQL managed instance. Both SQL managed instance and its partner have to use the same shared name."
+        },
+        "sourceMI": {
+          "type": "string",
+          "description": "The name of the SQL managed instance with this failover group role."
+        },
+        "partnerMI": {
+          "type": "string",
+          "description": "The name of the partner SQL managed instance."
+        },
+        "partnerMirroringURL": {
+          "type": "string",
+          "description": "The mirroring endpoint URL of the partner SQL managed instance."
+        },
+        "partnerMirroringCert": {
```

Comment:
>partnerMirroringCert [](http://example.com/codeflow?start=9&length=20)

mark it with x-ms-sceret

Reply:
Actually, this is just the public key, not considered a secret.